server/src/test/java/io/crate/integrationtests/ViewsITest.java

```diff
@@ -55,6 +58,36 @@ public void dropViews() {
         }
     }
 
+    @Test
+    @UseRandomizedSchema(random = false)
+    public void test_change_table_definition_information_schema_accessible() throws Exception {
+        execute("CREATE TABLE t01 (ts TIMESTAMP)");
+        execute("CREATE VIEW v01 AS SELECT ts::TIMESTAMP - '1 hour'::INTERVAL FROM t01");
+        execute("DROP TABLE t01");
+        execute("CREATE TABLE t01 (ts ARRAY(TIMESTAMP))");
+
+        // Verify that information schema is accessible
+        // and view definition has "corrupted, needs fix" hint as a comment.
+
+        execute("SELECT view_definition FROM information_schema.views where table_name = 'v01'");
+        assertThat(response).hasRows(
+            "/* Corrupted view, needs fix */ SELECT CAST(\"ts\" AS timestamp) - CAST('1 hour' AS interval)",
+            "FROM \"t01\""
+        );
```

Comment:
I probably misunderstand syntax of hasRows params but to me it looks like we expect 2 rows. But in runtime it's a single row with \n before FROM.

Found an [existing example](https://github.com/crate/crate/blob/master/server/src/test/java/io/crate/integrationtests/ViewsITest.java#L70-L73) where we do such assertion, also use multiple rows, probably it's fine.